core-bundle/src/Twig/Loader/ContaoFilesystemLoader.php

```diff
@@ -181,7 +181,11 @@ public function getCacheKey(string $name): string
     {
         $templateName = $this->getThemeTemplateName($name) ?? $name;
 
-        return parent::getCacheKey($templateName);
+        // We prefix the cache key to make sure templates from the default
+        // Symfony loader won't be reused. Otherwise, we cannot reliably
+        // differentiate when to apply our input encoding tolerant escaper
+        // filters (see #4623).
+        return 'c'.parent::getCacheKey($templateName);
```

Comment:
Is this safe or should we use `contao` instead?

Reply:
IMHO we just need to make sure there is a difference. And as the parent returns paths that start with a slash, a single char should be enough. 

But this gets hashed anyway, so I'm fine with any other prefix as well if you prefer. ☺️

Reply:
No, it‘s fine then.